CONTRIBUTING.md

```diff
@@ -0,0 +1,82 @@
+# Contributing with NovusCore
+
+Awesome! You want to help and contribute to NovusCore!
+
+You can contribute in different ways for NovusCore, either by opening bug reports, suggesting fixes for already open issues, proposing fixes 
+with Pull Requests (PR) or by suggesting new ideas.
+
+## What should I know before I start?
+
+1. First off, NovusCore has 3 projects which you can contribute to:
+
+Our [World of Warcraft 3.3.5 Branch](https://github.com/novuscore/NovusCore)
+
+Our [NovusCore-GMClient](https://github.com/novuscore/NovusCore-GMClient)
+
+Our [NovusCore-Client](https://github.com/novuscore/NovusCore-Client)
+
+Make sure you choose the correct project to contribute to.
+
+2. Do not open an issue or PR to ask a question, you will get a quicker response in our [Discord Channels](https://discordapp.com/invite/W8XTFJ?utm_source=Discord%20Widget&utm_medium=Connect).
+
+## Opening a Bug Report 📝
+
+1. First thing you should do is to make a quick search to see if the same or a similar issue is already opened.
+If so, just add your information to that report. It saves us a lot of time if we don't have to close duplicate reports.
+
+2. Always make sure you are running the latest revision of the core and database. If you are running something old chances are that your
+bug might already be fixed in the newer revisions.
+
+3. Never post more than **one** bug per issue report. For example; Quest A should have 1 report and Quest B should have another report unless
+they are directly related to each other. (Same goes for Creatures/Raids/Dungeons/Etc.)
+
+4. Never post a copy paste of a crash log in an issue.
+Use a service like [Github Gist](https://gist.github.com/) to paste your crash log and then link it in the issue with a description.
+
+5. If you have trouble writing in English you can use tools like Google Translate to translate from your native language, make sure to always add the original text in your native language (English + Native Language) as well since maybe someone else speaks that language as well.
```

Comment:
5. If you have trouble writing in English you can use tools like Google Translate to translate from your native language, make sure to always add the original text in your native language (English + Native Language) as well someone might just speak your language and be able to help you out.